.eslintrc.js

```diff
@@ -63,6 +64,7 @@ module.exports = {
           },
         ],
         '@typescript-eslint/consistent-type-imports': ['error'],
+        '@typescript-eslint/consistent-type-definitions': 'off',
```

Comment:
**suggestion:** we may add a comment to explain why we disable it for know. Adding a link to the PR comment is enough IMHO